rust-toolchain.toml

```
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "1.86.0"
```

Let's go with what's in Agave 2.2.

```suggestion
channel = "1.84.1"
```